.pre-commit-config.yaml

```diff
@@ -2,22 +2,22 @@ fail_fast: false
 default_language_version:
     python: python3
 default_stages:
-    - commit
-    - push
+    - pre-commit
```

Comment:
What is the difference here really?
This seems to be more common though, so fine to roll with this right @Zethson

Reply:
Whatever ^_^. Let's roll with this.